sst.config.ts

```diff
@@ -133,11 +142,7 @@ export const sstConfig: SSTConfig = {
       throw new Error('Please specify stage or set SST_USER env variable')
     }
 
-    if (
-      _input.stage &&
-      !availableStage.includes(_input.stage) &&
-      !_input.stage.startsWith('dev-')
-    ) {
+    if (_input.stage && !availableStage.includes(_input.stage)) {
```

Comment:
Do we want to disable dev stages to be deployed?

Reply:
Let me know if you have reasons to need that perhaps I can propose better alternative.

Reply:
No, looks good, just making sure it was not a bug